src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import './styles.css';
+import React from 'react';
+import Output from './output';
+
+const Calculator = () => (
+  <div className="calculator-grid">
+    <Output />
+    <button type="button">AC</button>
+    <button type="button">+/-</button>
+    <button type="button">%</button>
+    <button className="btn-orange" type="button">/</button>
+    <button type="button">7</button>
+    <button type="button">8</button>
+    <button type="button">9</button>
+    <button className="btn-orange" type="button">*</button>
```

Comment:
- [ ] [OPTIONAL] I think you should use an `x` instead of `*` for the multiplication symbol not to run into issues in the next milestone. 😎️

Reply:
@amejid Thanks for your suggestion. Will do that